birthday_automation/models/hr_employee.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, api
+from datetime import date
+
+class HrEmployee(models.Model):
+    #_name = 'birthday.handler'
+    _inherit = 'hr.employee'
+
+    @api.model
+    def send_birthday_email(self, employee_id):
+        email_template = self.env.ref('birthday_automation.birthday_email_template')
+        email_template.send_mail(employee_id)
+
+    @api.model
+    def _cron_birthday_handler(self):
+        
+        # Get current date
+        # Get filtered list of employees with a birthday on the current date
+        # Loop through each and send an email wishing a happy birthday
+
+        today = date.today()
+        employees = self.search([])
```

Comment:
It would be a better idea to first 

1. search for all employees that have birthday and work_email set
2. then filter those results using filtered